org.diasurgical.DevilutionX.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.diasurgical.DevilutionX
```

Comment:
the domain `diasurgical.org` seem to ne parked. does it belong to the developers?

Reply:
yes

Reply:
I'll change the domain to `io.github.diasurgical.DevilutionX`

Reply:
why? I actually own the diasurgical.org domain, that's what we use to validate for Google Play.
Use what we use elsewhere: `org.diasurgical.devilutionx`

Reply:
Let me know if you need to verify ownership or something.

Reply:
Oh, that's great then!

Reply:
```suggestion
app-id: org.diasurgical.devilutionx
```

Reply:
If you look at other app IDs, unless the app name is stylized as lower-case, it stays capitalized